ext/date/php_date.c

```diff
@@ -311,16 +311,18 @@ static void date_throw_uninitialized_error(zend_class_entry *ce)
 {
 	if (ce->type == ZEND_INTERNAL_CLASS) {
 		zend_throw_error(date_ce_date_object_error, "Object of type %s has not been correctly initialized by calling parent::__construct() in its constructor", ZSTR_VAL(ce->name));
```

Comment:
the `return;` was missing from here and from other places, therefore some tests broke. I guess these return fixes should be packported to lower branches, so I'll do that before merging this one.